Yoast/wordpress-seo
# Extract wp dependencies to their own files and register them.

## Summary

This PR can be summarized in the following changelog entry:

* Fixes console errors related to dependencies not being loaded.

## Test instructions
This PR can be tested by following these steps:

* The metabox should load and work properly when Gutenberg is loaded as a plugin, as part of core 5.0 or when not loaded at all.
* The same should apply for all other scripts ( dashboard widget, configuration wizard, term scraper, help-center, Google Search Console etc. ).
* Make sure no Gutenberg scripts are loaded on the widget.php page. See https://github.com/Yoast/wordpress-seo/issues/10871
* Test the following setups (also in combination with each other): 
    - WordPress 4.9.8
    - WordPress 5.0 beta 3 (or the newest available beta)
    - classic editor plugin
    - default classic editor
    - latest released Gutenberg version (+ latest Gutenberg RC, if available)
   - Please also test with the ways to disable/enable Gutenberg mentioned in this post: https://make.wordpress.org/core/2018/10/30/block-editor-filters/. Related issue: https://github.com/Yoast/wordpress-seo/issues/11519 **If this PR breaks on these scenarios, please create another issue and don't let it block the merging of this PR.**

## Quality assurance


Fixes #11418 
Fixes #10871
Fixes #11131
Fixes #11409
Fixes #11519
